tests/numerics/fv/test_tpsa.py

```diff
@@ -1269,11 +1269,12 @@ def _assemble_matrices(
     if g.dim == 2:
         n_rot_face = g.num_faces
         n_rot_cell = g.num_cells
-        div_rot = pp.fvutils.scalar_divergence(g)
+        div_rot = g.divergence(dim=1)
```

Comment:
Not accessed. Is that cause for concern?

Reply:
Yes it is (verified by manual inspection). There were bugs here and below, though, and I found a better way to structure the code.